client-misk-dynamodb/src/main/kotlin/app/cash/backfila/client/misk/dynamodb/DynamoDbBackfill.kt

```diff
@@ -0,0 +1,39 @@
+package app.cash.backfila.client.misk.dynamodb
+
+import app.cash.backfila.client.misk.Backfill
+import app.cash.backfila.client.misk.BackfillConfig
+
+abstract class DynamoDbBackfill<I : Any, P : Any> : Backfill {
+
+  abstract val itemType: Class<I>
```

Comment:
couldn't you use the same trick to get this as in the hibernate one?

Reply:
good idea

Reply:
Yeah of course. Done